### PR TITLE
Updates for new Telemetry, Dome door position and DIMM seeing.

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: bin/telemetry_sim_ATHS [hunk 1]
```diff
@@ -71,6 +71,15 @@ def cmdline():
     parser.add_argument("--imagetag", action="store", default="ElGordo", type=str,
                         help="The imagetag of the observation.")
 
+    parser.add_argument("--seeing", action="store", default=1.2, type=float,
+                        help="The Seeing from the DIMM monitor (arcsec).")
+    parser.add_argument("--domeaz", action="store", default=45, type=float,
+                        help="The Dome AZ position (deg).")
+    parser.add_argument("--shutterlow", action="store", default=100, type=float,
+                        help="Dome Dropout Door Opening Percertage.")
+    parser.add_argument("--shutterupp", action="store", default=100, type=float,
```
Comment (patrickingraham) [inline bin/telemetry_sim_ATHS:80]: default should be zero (closed)

File: bin/telemetry_sim_ATHS [hunk 2]
```diff
@@ -71,6 +71,15 @@ def cmdline():
     parser.add_argument("--imagetag", action="store", default="ElGordo", type=str,
                         help="The imagetag of the observation.")
 
+    parser.add_argument("--seeing", action="store", default=1.2, type=float,
+                        help="The Seeing from the DIMM monitor (arcsec).")
+    parser.add_argument("--domeaz", action="store", default=45, type=float,
+                        help="The Dome AZ position (deg).")
+    parser.add_argument("--shutterlow", action="store", default=100, type=float,
```
Comment (patrickingraham) [inline bin/telemetry_sim_ATHS]: default should be zero (closed)

**Review by patrickingraham — APPROVED** [review]
Please make the minor changes before merging

File: bin/telemetry_sim_ATHS (hunk 2 above)
Comment (menanteau) [reply bin/telemetry_sim_ATHS]: Done, thanks.

**Pull request merged**